shared/domains/chelonia/chelonia.js

```diff
@@ -270,9 +271,9 @@ export default (sbp('sbp/selectors/register', {
   },
   // 'chelonia/contract' - selectors related to injecting remote data and monitoring contracts
   // TODO: add an optional parameter to "retain" the contract (see #828)
-  'chelonia/contract/sync': function (contractIDs: string | string[]): Promise<*> {
+  'chelonia/contract/sync': async function (contractIDs: string | string[]): Promise<*> {
     const listOfIds = typeof contractIDs === 'string' ? [contractIDs] : contractIDs
-    return Promise.all(listOfIds.map(contractID => {
+    await Promise.all(listOfIds.map(contractID => {
```

Comment:
Could you revert these changes in case having the return value ends up useful?

Reply:
Okay.